internal/archive/archive.go

```diff
@@ -270,6 +325,11 @@ func (index *ubuntuIndex) fetchIndex() error {
 	return nil
 }
 
+func (index *ubuntuIndex) supportsArch(arch string) bool {
+	supportedArches := strings.Fields(index.release.Get("Architectures"))
```

Comment:
Let's try to use `strings.Contains` since the architectures are space separated and unique.